build.proj

```diff
@@ -33,6 +33,8 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
+
+    <DotnetCliBuildDirectory>$(MSBuildThisFileDirectory)/build_projects/dotnet-cli-build/</DotnetCliBuildDirectory>
```

Comment:
This is inconsistent with MSBuild built-in properties but I'll make the change for now and do a consistency pass later.